apis/v1alpha1/gateway_types.go

```diff
@@ -97,11 +93,11 @@ type GatewaySpec struct {
 	// behavior can depend on the GatewayClass. If a value is set
 	// in the spec and the requested address is invalid, the
 	// GatewayClass MUST indicate this in the associated entry in
-	// GatewayStatus.Listeners.
+	// GatewayStatus.Addresses.
```

Comment:
This paragraph is trying to say that if an address can't be assigned, the controller should raise a `ConditionInvalidAddress`, but it's mangling it badly.